openjdk/build.gradle

```diff
@@ -153,19 +156,17 @@ model {
 
     tasks {
         $.binaries.withType(SharedLibraryBinarySpec).each {
-            def archName = it.targetPlatform.architecture.name.replaceAll('-', '_')
             def source = it.sharedLibraryFile
 
             // Copies the native library to a resource location that will be included in the jar.
-            task "copyNativeLib${archName}"(type: Copy) {
+            task "copySharedNativeLib${osdetector.arch}"(type: Copy) {
```

Comment:
`osdetector.arch` is always the same answer and not the build target's architecture. This is used to unify the 32-bit and 64-bit libraries.